src/commonMain/kotlin/com/github/quillraven/fleks/entity.kt

```diff
@@ -20,6 +20,10 @@ class EntityHookContext(
     inline operator fun <reified T : Component<*>> Entity.get(type: ComponentType<T>): T {
         return compService.holder(type)[this]
     }
+
+    inline fun <reified T : Component<*>> Entity.contains(componentType: ComponentType<T>): Boolean {
```

Comment:
I suggest to make it an operator fun then you can also use the `in` syntax if you want.

Reply:
So that means removing the `inline`ing? Didn't know that. I need to check the `in` syntax in my code...

Reply:
Ah now I got it ;)